### PR TITLE
chore: py production scripts are bazelified

### LINKED ISSUE #13415: python scripts are bazelified

### PR DESCRIPTION
**Pull request by nstng**
## Summary

This change creates `py_binary` targets for all python scripts that are packaged into a magma debian build. This is a preparation for #13416.

## Test Plan

* make sure that all relevant scripts are bazelified
  * a recent magma debian package can be downloaded from https://artifactory.magmacore.org/artifactory/debian-test/pool/focal-ci/ and the output of `dpkg -c magma_foo.deb | grep "/usr/local/bin" | grep "\.py"` can be compared with the list of bazelified scripts.
* make sure that the targets do not have missing dependencies - basically run them and see that there are no ModuleNotFound issues [1]
* verify that the scripts behave the same as for the make build - this is the hard part, because almost all scripts do not run ootb on the magma vm. This needs a vm where `lte/gateway $ make run` was executed and magma is running.
  * run make scripts
    * `sudo source /etc/environment`
    * `sudo chmod a+r /var/log/syslog`
    * link scripts to `/usr/local/bin` [2]
    * run a script, e.g., `agw_health_cli.py`
  * run bazel scripts
    * do a fresh ssh login (in order to revert `source /etc/environment` from above)
    * fyi: we need to also link everything because some scripts call other scripts via subprocess calls
    * build all binaries [3]
    * link all binaries [4]
    * run a script, e.g., `agw_health_cli.py`

[1]
```
bazel run lte/gateway/python/scripts:agw_health_cli
bazel run lte/gateway/python/scripts:config_stateless_agw
bazel run lte/gateway/python/scripts:cpe_monitoring_cli
bazel run lte/gateway/python/scripts:create_oai_certs
bazel run lte/gateway/python/scripts:dp_probe_cli
bazel run lte/gateway/python/scripts:enodebd_cli
bazel run lte/gateway/python/scripts:fake_user
bazel run lte/gateway/python/scripts:feg_hello_cli
bazel run lte/gateway/python/scripts:generate_dnsd_config
bazel run lte/gateway/python/scripts:generate_oai_config
bazel run lte/gateway/python/scripts:ha_cli
bazel run lte/gateway/python/scripts:hello_cli
bazel run lte/gateway/python/scripts:icmpv6
bazel run lte/gateway/python/scripts:mobility_cli
bazel run lte/gateway/python/scripts:mobility_dhcp_cli
bazel run lte/gateway/python/scripts:ocs_cli
bazel run lte/gateway/python/scripts:packet_ryu_cli
bazel run lte/gateway/python/scripts:packet_tracer_cli
bazel run lte/gateway/python/scripts:pcrf_cli
bazel run lte/gateway/python/scripts:pipelined_cli
bazel run lte/gateway/python/scripts:policydb_cli
bazel run lte/gateway/python/scripts:s6a_proxy_cli
bazel run lte/gateway/python/scripts:s6a_service_cli
bazel run lte/gateway/python/scripts:session_manager_cli
bazel run lte/gateway/python/scripts:sgs_cli
bazel run lte/gateway/python/scripts:sms_cli
bazel run lte/gateway/python/scripts:spgw_service_cli
bazel run lte/gateway/python/scripts:state_cli
bazel run lte/gateway/python/scripts:subscriber_cli
bazel run lte/gateway/python/scripts:user_trace_cli
bazel run orc8r/gateway/python/scripts:checkin_cli
bazel run orc8r/gateway/python/scripts:ctraced_cli
bazel run orc8r/gateway/python/scripts:directoryd_cli
bazel run orc8r/gateway/python/scripts:generate_fluent_bit_config
bazel run orc8r/gateway/python/scripts:generate_lighttpd_config
bazel run orc8r/gateway/python/scripts:generate_nghttpx_config
bazel run orc8r/gateway/python/scripts:generate_service_config
bazel run orc8r/gateway/python/scripts:health_cli
bazel run orc8r/gateway/python/scripts:magma_conditional_service
bazel run orc8r/gateway/python/scripts:magma_get_config
bazel run orc8r/gateway/python/scripts:magmad_cli
bazel run orc8r/gateway/python/scripts:service303_cli
bazel run orc8r/gateway/python/scripts:service_util
bazel run orc8r/gateway/python/scripts:show_gateway_info
bazel run orc8r/gateway/python/scripts:traffic_cli
bazel run //lte/gateway/python/load_tests:loadtest_mobilityd
bazel run //lte/gateway/python/load_tests:loadtest_pipelined
bazel run //lte/gateway/python/load_tests:loadtest_sessiond
bazel run //lte/gateway/python/load_tests:loadtest_subscriberdb
```

[2]
```
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/agw_health_cli.py /usr/local/bin/agw_health_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/config_stateless_agw.py /usr/local/bin/config_stateless_agw.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/cpe_monitoring_cli.py /usr/local/bin/cpe_monitoring_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/create_oai_certs.py /usr/local/bin/create_oai_certs.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/dp_probe_cli.py /usr/local/bin/dp_probe_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/enodebd_cli.py /usr/local/bin/enodebd_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/fake_user.py /usr/local/bin/fake_user.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/feg_hello_cli.py /usr/local/bin/feg_hello_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/generate_dnsd_config.py /usr/local/bin/generate_dnsd_config.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/generate_oai_config.py /usr/local/bin/generate_oai_config.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/ha_cli.py /usr/local/bin/ha_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/hello_cli.py /usr/local/bin/hello_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/icmpv6.py /usr/local/bin/icmpv6.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/mobility_cli.py /usr/local/bin/mobility_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/mobility_dhcp_cli.py /usr/local/bin/mobility_dhcp_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/ocs_cli.py /usr/local/bin/ocs_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/packet_ryu_cli.py /usr/local/bin/packet_ryu_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/packet_tracer_cli.py /usr/local/bin/packet_tracer_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/pcrf_cli.py /usr/local/bin/pcrf_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/pipelined_cli.py /usr/local/bin/pipelined_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/policydb_cli.py /usr/local/bin/policydb_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/s6a_proxy_cli.py /usr/local/bin/s6a_proxy_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/s6a_service_cli.py /usr/local/bin/s6a_service_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/session_manager_cli.py /usr/local/bin/session_manager_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/sgs_cli.py /usr/local/bin/sgs_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/sms_cli.py /usr/local/bin/sms_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/spgw_service_cli.py /usr/local/bin/spgw_service_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/state_cli.py /usr/local/bin/state_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/subscriber_cli.py /usr/local/bin/subscriber_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/scripts/user_trace_cli.py /usr/local/bin/user_trace_cli.py
sudo ln -sf /home/vagrant/magma/orc8r/gateway/python/scripts/checkin_cli.py /usr/local/bin/checkin_cli.py
sudo ln -sf /home/vagrant/magma/orc8r/gateway/python/scripts/ctraced_cli.py /usr/local/bin/ctraced_cli.py
sudo ln -sf /home/vagrant/magma/orc8r/gateway/python/scripts/directoryd_cli.py /usr/local/bin/directoryd_cli.py
sudo ln -sf /home/vagrant/magma/orc8r/gateway/python/scripts/generate_fluent_bit_config.py /usr/local/bin/generate_fluent_bit_config.py
sudo ln -sf /home/vagrant/magma/orc8r/gateway/python/scripts/generate_lighttpd_config.py /usr/local/bin/generate_lighttpd_config.py
sudo ln -sf /home/vagrant/magma/orc8r/gateway/python/scripts/generate_nghttpx_config.py /usr/local/bin/generate_nghttpx_config.py
sudo ln -sf /home/vagrant/magma/orc8r/gateway/python/scripts/generate_service_config.py /usr/local/bin/generate_service_config.py
sudo ln -sf /home/vagrant/magma/orc8r/gateway/python/scripts/health_cli.py /usr/local/bin/health_cli.py
sudo ln -sf /home/vagrant/magma/orc8r/gateway/python/scripts/magma_conditional_service.py /usr/local/bin/magma_conditional_service.py
sudo ln -sf /home/vagrant/magma/orc8r/gateway/python/scripts/magma_get_config.py /usr/local/bin/magma_get_config.py
sudo ln -sf /home/vagrant/magma/orc8r/gateway/python/scripts/magmad_cli.py /usr/local/bin/magmad_cli.py
sudo ln -sf /home/vagrant/magma/orc8r/gateway/python/scripts/service303_cli.py /usr/local/bin/service303_cli.py
sudo ln -sf /home/vagrant/magma/orc8r/gateway/python/scripts/service_util.py /usr/local/bin/service_util.py
sudo ln -sf /home/vagrant/magma/orc8r/gateway/python/scripts/show_gateway_info.py /usr/local/bin/show_gateway_info.py
sudo ln -sf /home/vagrant/magma/orc8r/gateway/python/scripts/traffic_cli.py /usr/local/bin/traffic_cli.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/load_tests/loadtest_mobilityd.py /usr/local/bin/loadtest_mobilityd.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/load_tests/loadtest_pipelined.py /usr/local/bin/loadtest_pipelined.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/load_tests/loadtest_sessiond.py /usr/local/bin/loadtest_sessiond.py
sudo ln -sf /home/vagrant/magma/lte/gateway/python/load_tests/loadtest_subscriberdb.py /usr/local/bin/loadtest_subscriberdb.py
```

[3]
```
bazel build lte/gateway/python/scripts:agw_health_cli lte/gateway/python/scripts:config_stateless_agw lte/gateway/python/scripts:cpe_monitoring_cli lte/gateway/python/scripts:create_oai_certs lte/gateway/python/scripts:dp_probe_cli lte/gateway/python/scripts:enodebd_cli lte/gateway/python/scripts:fake_user lte/gateway/python/scripts:feg_hello_cli lte/gateway/python/scripts:generate_dnsd_config lte/gateway/python/scripts:generate_oai_config lte/gateway/python/scripts:ha_cli lte/gateway/python/scripts:hello_cli lte/gateway/python/scripts:icmpv6 lte/gateway/python/scripts:mobility_cli lte/gateway/python/scripts:mobility_dhcp_cli lte/gateway/python/scripts:ocs_cli lte/gateway/python/scripts:packet_ryu_cli lte/gateway/python/scripts:packet_tracer_cli lte/gateway/python/scripts:pcrf_cli lte/gateway/python/scripts:pipelined_cli lte/gateway/python/scripts:policydb_cli lte/gateway/python/scripts:s6a_proxy_cli lte/gateway/python/scripts:s6a_service_cli lte/gateway/python/scripts:session_manager_cli lte/gateway/python/scripts:sgs_cli lte/gateway/python/scripts:sms_cli lte/gateway/python/scripts:spgw_service_cli lte/gateway/python/scripts:state_cli lte/gateway/python/scripts:subscriber_cli lte/gateway/python/scripts:user_trace_cli orc8r/gateway/python/scripts:checkin_cli orc8r/gateway/python/scripts:ctraced_cli orc8r/gateway/python/scripts:directoryd_cli orc8r/gateway/python/scripts:generate_fluent_bit_config orc8r/gateway/python/scripts:generate_lighttpd_config orc8r/gateway/python/scripts:generate_nghttpx_config orc8r/gateway/python/scripts:generate_service_config orc8r/gateway/python/scripts:health_cli orc8r/gateway/python/scripts:magma_conditional_service orc8r/gateway/python/scripts:magma_get_config orc8r/gateway/python/scripts:magmad_cli orc8r/gateway/python/scripts:service303_cli orc8r/gateway/python/scripts:service_util orc8r/gateway/python/scripts:show_gateway_info orc8r/gateway/python/scripts:traffic_cli //lte/gateway/python/load_tests:loadtest_mobilityd //lte/gateway/python/load_tests:loadtest_pipelined //lte/gateway/python/load_tests:loadtest_sessiond //lte/gateway/python/load_tests:loadtest_subscriberdb
```

[4]
```
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/agw_health_cli /usr/local/bin/agw_health_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/config_stateless_agw /usr/local/bin/config_stateless_agw.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/cpe_monitoring_cli /usr/local/bin/cpe_monitoring_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/create_oai_certs /usr/local/bin/create_oai_certs.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/dp_probe_cli /usr/local/bin/dp_probe_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/enodebd_cli /usr/local/bin/enodebd_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/fake_user /usr/local/bin/fake_user.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/feg_hello_cli /usr/local/bin/feg_hello_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/generate_dnsd_config /usr/local/bin/generate_dnsd_config.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/generate_oai_config /usr/local/bin/generate_oai_config.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/ha_cli /usr/local/bin/ha_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/hello_cli /usr/local/bin/hello_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/icmpv6 /usr/local/bin/icmpv6.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/mobility_cli /usr/local/bin/mobility_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/mobility_dhcp_cli /usr/local/bin/mobility_dhcp_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/ocs_cli /usr/local/bin/ocs_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/packet_ryu_cli /usr/local/bin/packet_ryu_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/packet_tracer_cli /usr/local/bin/packet_tracer_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/pcrf_cli /usr/local/bin/pcrf_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/pipelined_cli /usr/local/bin/pipelined_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/policydb_cli /usr/local/bin/policydb_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/s6a_proxy_cli /usr/local/bin/s6a_proxy_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/s6a_service_cli /usr/local/bin/s6a_service_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/session_manager_cli /usr/local/bin/session_manager_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/sgs_cli /usr/local/bin/sgs_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/sms_cli /usr/local/bin/sms_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/spgw_service_cli /usr/local/bin/spgw_service_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/state_cli /usr/local/bin/state_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/subscriber_cli /usr/local/bin/subscriber_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/user_trace_cli /usr/local/bin/user_trace_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/orc8r/gateway/python/scripts/checkin_cli /usr/local/bin/checkin_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/orc8r/gateway/python/scripts/ctraced_cli /usr/local/bin/ctraced_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/orc8r/gateway/python/scripts/directoryd_cli /usr/local/bin/directoryd_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/orc8r/gateway/python/scripts/generate_fluent_bit_config /usr/local/bin/generate_fluent_bit_config.py
sudo ln -sf /home/vagrant/magma/bazel-bin/orc8r/gateway/python/scripts/generate_lighttpd_config /usr/local/bin/generate_lighttpd_config.py
sudo ln -sf /home/vagrant/magma/bazel-bin/orc8r/gateway/python/scripts/generate_nghttpx_config /usr/local/bin/generate_nghttpx_config.py
sudo ln -sf /home/vagrant/magma/bazel-bin/orc8r/gateway/python/scripts/generate_service_config /usr/local/bin/generate_service_config.py
sudo ln -sf /home/vagrant/magma/bazel-bin/orc8r/gateway/python/scripts/health_cli /usr/local/bin/health_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/orc8r/gateway/python/scripts/magma_conditional_service /usr/local/bin/magma_conditional_service.py
sudo ln -sf /home/vagrant/magma/bazel-bin/orc8r/gateway/python/scripts/magma_get_config /usr/local/bin/magma_get_config.py
sudo ln -sf /home/vagrant/magma/bazel-bin/orc8r/gateway/python/scripts/magmad_cli /usr/local/bin/magmad_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/orc8r/gateway/python/scripts/service303_cli /usr/local/bin/service303_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/orc8r/gateway/python/scripts/service_util /usr/local/bin/service_util.py
sudo ln -sf /home/vagrant/magma/bazel-bin/orc8r/gateway/python/scripts/show_gateway_info /usr/local/bin/show_gateway_info.py
sudo ln -sf /home/vagrant/magma/bazel-bin/orc8r/gateway/python/scripts/traffic_cli /usr/local/bin/traffic_cli.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/load_tests/loadtest_mobilityd /usr/local/bin/loadtest_mobilityd.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/load_tests/loadtest_pipelined /usr/local/bin/loadtest_pipelined.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/load_tests/loadtest_sessiond /usr/local/bin/loadtest_sessiond.py
sudo ln -sf /home/vagrant/magma/bazel-bin/lte/gateway/python/load_tests/loadtest_subscriberdb /usr/local/bin/loadtest_subscriberdb.py
```

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by vktng — COMMENTED** [review]
I have verified that all script in the magma deb are bazelified.
I have run all bazel targets and they do not have ModuleNotFound issues.
I have done the documented preparations for Make execution. The scripts have problems when executed with sudo (and this is sometimes needed). Some examples:
`sudo create_oai_certs.py`
```
Traceback (most recent call last):
File "/usr/local/bin/create_oai_certs.py", line 22, in <module>
	import envoy
ImportError: No module named envoy
```

`sudo dp_probe_cli.py -i 5 -d UL stats`
```
sudo: dp_probe_cli.py: command not found
```

Scripts with similar problems:

- config_stateless_agw
- create_oai_certs
- dp_probe_cli
- generate_dnsd_config
- packet_tracer_cli
- pipelined_cli
- generate_fluent_bit_config
- generate_nghttpx_config

Other then the scripts that need sudo the output of the Bazel counterparts were the same, except in one case:
`loadtest_pipelined.py activate_flows --num_of_ues 10 --import_path /`
With Bazel:
```
Preparing activate_flows load test...
QOS Enabled
open lte/protos/lte/protos/pipelined.proto: no such file or directory
Done
```
With Make:
```
Preparing activate_flows load test...
QOS Enabled
Done
```

There are also some scripts that are just broken anyway, for example:
`mobility_dhcp_cli.py`
```
		Traceback (most recent call last):
		  File "/home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/mobility_dhcp_cli.runfiles/__main__/lte/gateway/python/scripts/mobility_dhcp_cli.py", line 208, in <module>
		    main()
		  File "/home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/mobility_dhcp_cli.runfiles/__main__/lte/gateway/python/scripts/mobility_dhcp_cli.py", line 125, in main
		    cli = DhcpClientCLI()
		  File "/home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/mobility_dhcp_cli.runfiles/__main__/lte/gateway/python/scripts/mobility_dhcp_cli.py", line 37, in __init__
		    self.dhcp_client_state = store.MacToIP()  # mac => DHCP_State
		TypeError: __init__() missing 1 required positional argument: 'client'
```

**Comment by vktng** [comment]
> I have verified that all script in the magma deb are bazelified. I have run all bazel targets and they do not have ModuleNotFound issues. I have done the documented preparations for Make execution. The scripts have problems when executed with sudo (and this is sometimes needed). Some examples: `sudo create_oai_certs.py`
> 
> ```
> Traceback (most recent call last):
> File "/usr/local/bin/create_oai_certs.py", line 22, in <module>
> 	import envoy
> ImportError: No module named envoy
> ```
> 
> `sudo dp_probe_cli.py -i 5 -d UL stats`
> 
> ```
> sudo: dp_probe_cli.py: command not found
> ```
> 
> Scripts with similar problems:
> 
>     * config_stateless_agw
> 
>     * create_oai_certs
> 
>     * dp_probe_cli
> 
>     * generate_dnsd_config
> 
>     * packet_tracer_cli
> 
>     * pipelined_cli
> 
>     * generate_fluent_bit_config
> 
>     * generate_nghttpx_config
> 
> 
> Other then the scripts that need sudo the output of the Bazel counterparts were the same, except in one case: `loadtest_pipelined.py activate_flows --num_of_ues 10 --import_path /` With Bazel:
> 
> ```
> Preparing activate_flows load test...
> QOS Enabled
> open lte/protos/lte/protos/pipelined.proto: no such file or directory
> Done
> ```
> 
> With Make:
> 
> ```
> Preparing activate_flows load test...
> QOS Enabled
> Done
> ```
> 
> There are also some scripts that are just broken anyway, for example: `mobility_dhcp_cli.py`
> 
> ```
> 		Traceback (most recent call last):
> 		  File "/home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/mobility_dhcp_cli.runfiles/__main__/lte/gateway/python/scripts/mobility_dhcp_cli.py", line 208, in <module>
> 		    main()
> 		  File "/home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/mobility_dhcp_cli.runfiles/__main__/lte/gateway/python/scripts/mobility_dhcp_cli.py", line 125, in main
> 		    cli = DhcpClientCLI()
> 		  File "/home/vagrant/magma/bazel-bin/lte/gateway/python/scripts/mobility_dhcp_cli.runfiles/__main__/lte/gateway/python/scripts/mobility_dhcp_cli.py", line 37, in __init__
> 		    self.dhcp_client_state = store.MacToIP()  # mac => DHCP_State
> 		TypeError: __init__() missing 1 required positional argument: 'client'
> ```

To solve the sudo problems I executed the scripts with the following command:
`sudo -E env "PATH=$PATH" some_script.py [args]`
This way I verified that all the scripts that require sudo behave the same with Make as with Bazel if we preserve the `PATH` variable.
The only difference that remains is with `loadtest_pipelined.py` as mentioned before.

**Comment by nstng** [comment]
> To solve the sudo problems I executed the scripts with the following command: `sudo -E env "PATH=$PATH" some_script.py [args]` This way I verified that all the scripts that require sudo behave the same with Make as with Bazel if we preserve the `PATH` variable. The only difference that remains is with `loadtest_pipelined.py` as mentioned before.

I think `sudo source /etc/environment` should have been solved this - I forgot the `sudo` in the description (added).

**Comment by nstng** [comment]
The status of the scripts in a production environment based on a magma .deb setup is described in #13606.

The current review shows that the only difference in behavior can be found in `loadtest_pipelined.py` - this script does not work in a production environment (see #13606).

I recommend to merge this state (if there are no further issues found) and adapt bazelified scripts when working on issues originating from #13606.

File: lte/gateway/python/magma/mobilityd/BUILD.bazel [hunk 1]
```diff
@@ -44,36 +44,69 @@ py_library(
...
+)
+
+py_library(
+    name = "dhcp_desc",
+    srcs = [
+        "dhcp_desc.py",
+        "mac.py",
+    ],
+    visibility = ["//visibility:public"],
+    deps = [
```
Comment (vktng) [inline lte/gateway/python/magma/mobilityd/BUILD.bazel]: Empty deps.

File: lte/gateway/python/magma/mobilityd/BUILD.bazel [hunk 2]
```diff
@@ -44,36 +44,69 @@ py_library(
...
+
+py_library(
+    name = "serialize_utils",
+    srcs = [
+        "ip_descriptor.py",
+        "serialize_utils.py",
+        "utils.py",
+    ],
+    visibility = ["//visibility:public"],
+    deps = [
```
Comment (vktng) [inline lte/gateway/python/magma/mobilityd/BUILD.bazel]: Can be formatted to use 1 line.

File: lte/gateway/python/scripts/BUILD.bazel [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+        "//orc8r/gateway/python/magma/common:rpc_utils",
+    ],
+)
+
+py_binary(
+    name = "generate_dnsd_config",
+    srcs = ["generate_dnsd_config.py"],
+    imports = [
+        ORC8R_ROOT,
+        "{}orc8r/gateway/python/scripts".format(MAGMA_ROOT),
```
Comment (vktng) [inline lte/gateway/python/scripts/BUILD.bazel]: Could be a variable like the other roots.

File: lte/gateway/python/scripts/BUILD.bazel [hunk 4]
```diff
@@ -0,0 +1,429 @@
...
+        requirement("scapy"),
+    ],
+)
+
+py_binary(
+    name = "packet_tracer_cli",
+    srcs = ["packet_tracer_cli.py"],
+    imports = [ORC8R_ROOT],
+    legacy_create_init = False,
+    deps = [
```
Comment (vktng) [inline lte/gateway/python/scripts/BUILD.bazel]: Can be formatted to use 1 line.

**Review by vktng — CHANGES_REQUESTED** [review]
Please address the comments regarding formatting. Otherwise LGTM.

**Comment by nstng** [comment]
review changes addressed.

File: lte/gateway/python/scripts/BUILD.bazel [hunk 5]
```diff
@@ -0,0 +1,429 @@
...
+        "//orc8r/gateway/python/scripts:generate_service_config",
+    ],
+)
+
+py_binary(
+    name = "generate_oai_config",
+    srcs = ["generate_oai_config.py"],
+    imports = [
+        ORC8R_ROOT,
+        "{}orc8r/gateway/python/scripts".format(MAGMA_ROOT),
```
Comment (vktng) [inline lte/gateway/python/scripts/BUILD.bazel]: Please use the SCRIPTS_ROOT variable.

Comment (nstng) [reply lte/gateway/python/scripts/BUILD.bazel]: oO thanks for finding this one

Comment (nstng) [reply lte/gateway/python/scripts/BUILD.bazel]: pushed

**Review by vktng — APPROVED** [review]
LGTM.

**Pull request merged**